docs/src/index.scss

```diff
@@ -6,23 +6,23 @@
 // only bring in the partials we need, doing something similar to ncids.scss
 
 // token overrides
-@import '@nciocpl/ncids-css/scss/base/settings/settings-color';
+@import '@nciocpl/ncids-css/scss/settings/settings-color';
```

Comment:
Is this correct here since it is in required?

Reply:
nope, will be removed